README.md

```diff
@@ -154,7 +154,7 @@ cp kubectl-crossplane /usr/local/bin
 
 ```console
 # Check the latest version available in https://cloud.upbound.io/registry/upbound/platform-ref-azure
-PLATFORM_VERSION=v0.1.0
+PLATFORM_VERSION=v0.3.0
```

Comment:
thanks for bumping this version ahead of an upcoming release - there's also 1 more instance further down in the README, can you update that one too?

```
VERSION_TAG=v0.1.0
```

Reply:
I deliberately skipped that line. Because it belongs to [Customize for your Organization](https://github.com/upbound/platform-ref-azure#customize-for-your-organization) section which explains how people can push their own packages to their own repos. Do you think it makes sense to keep that version up-to-date with the upstream repo version?

Reply:
Ah good observation - yeah that logic makes sense, I'm fine with that!